src/org/rascalmpl/library/ParseTree.rsc

```diff
@@ -310,7 +310,7 @@ data Symbol
 data Symbol // <19>
      = \conditional(Symbol symbol, set[Condition] conditions);
 
-public bool subtype(Symbol::\sort(_), Symbol::\adt("Tree", _)) = true;
```

Comment:
ParseTree is one of the most visible modules in the library; by using `public` everywhere it is priming our users to do the same, while we decided that `public` was the default. Over the course of the next months, I'll be removing those modifiers from the library modules.

Reply:
ah right, that's why this sneaked in ;)

Reply:
yes, a guilty pleasure.